src/main/webapp/css/srp_style.css

```diff
@@ -36,34 +36,61 @@ h1 {
 }
 
 .card {
-  margin-bottom: 20px;
-  padding: 24px;
+  margin: 20px 0px;
+  padding: 24px 16px 24px 24px;
   box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 3px 1px -2px rgba(0, 0, 0, 0.12), 0 1px 5px 0 rgba(0, 0, 0, 0.2);
   transition: box-shadow .25s;
   border-radius: 2px;
   width: 400px;
+  max-height: 260px;
+  overflow: scroll;
+  font-size: 16px;
+  scrollbar-color: grey #FAFAFA;
 }
 
-.card .card-title {
-  font: 300 20px Roboto, Arial, sans-serif;
+.card::-webkit-scrollbar {
+  width: 8px;
+  height: 8px;
+  background-color: #FAFAFA;
+}
+
+.card::-webkit-scrollbar-thumb, pre::-webkit-scrollbar-thumb {
+  background: lightgrey;
+  border-radius: 10px;
 }
 
-.card .card-description {
-  font: bold 16px Roboto, Arial, sans-serif;
-  margin-top: 20px;
+.card::-webkit-scrollbar-thumb:hover, pre::-webkit-scrollbar-thumb:hover {
+  background: grey;
+}
+
+.card .card-title {
+  font: 300 20px Roboto, Arial, sans-serif;
```

Comment:
Same here, font is already inherited.

Reply:
is there a way to combine font size and weight without including the font family at the end?

Reply:
Nope.

https://developer.mozilla.org/en-US/docs/Web/CSS/font
If font is specified as a shorthand for several font-related properties, then:

    it must include values for:
        <font-size>
        <font-family>